features/git-extract/cherry_pick_conflict/with_remote_origin/with_open_changes.feature

```diff
@@ -85,6 +80,13 @@ Feature: git extract: resolving conflicts between main branch and extracted comm
       |          |                  | refactor commit | conflicting_file |
       | refactor | local and remote | main commit     | conflicting_file |
       |          |                  | refactor commit | conflicting_file |
+    And now I have the following committed files
+      | BRANCH   | NAME             | CONTENT          |
+      | main     | conflicting_file | main content     |
+      | feature  | conflicting_file | refactor content |
+      | feature  | feature_file     |                  |
+      | refactor | conflicting_file | resolved content |
```

Comment:
Interesting idea. In this case we should create separate steps for verifying commits and verifying commits and files

```cucumber
    Then now I have the following commits
      | BRANCH   | LOCATION         | MESSAGE     | 
      | main     | local and remote | main commit | 
      (does not verify the files in the commits)

    Then now I have the following commits and files
      | BRANCH  | LOCATION         | MESSAGE        | FILE NAME        | FILE CONTENT     |
      | main    | local and remote | main commit    | conflicting_file | resolved content |
      | feature | local and remote | feature commit |                  |                  |
      (verifies the file content, backfilling with default values if necessary)